worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/Flusher.scala

```diff
@@ -141,13 +147,17 @@ private[worker] class LocalFlusher(
     maxComponents: Int,
     val mountPoint: String,
     val diskType: StorageInfo.Type,
-    timeWindow: TimeWindow) extends Flusher(
+    timeWindow: TimeWindow,
+    reuseCopyBuffer: Boolean,
```

Comment:
You should set `reuseCopyBuffer` to false if you don't want the local flusher to get involved with the unnecessary memory copy operations.